code/modules/projectiles/ammunition/_ammunition.dm

```diff
@@ -29,6 +29,8 @@
 	/// If true, the casing's sprite will automatically be transformed in Initialize().
 	/// Disable for things like rockets or other heavy ammunition that should only appear right-side up.
 	var/auto_rotate = TRUE
+	/// If you dont want to bullets to randomly change position on spawn. For mapping.
+	var/auto_scatter = TRUE
```

Comment:
I can't say for certain this is why, but I think it's done on init instead of mapload since these things are still being spawned throughout the game. 

it'd look a bit odd if you printed a box of bullets mid-round after the mapload, dumped it out and all the bullets came out in the exact same position.

I could be misunderstanding how mapload works though.